src/screens/OParl/OParlPersonsScreen.tsx

```diff
@@ -158,19 +171,20 @@ export const OParlPersonsScreen = ({ navigation }: Props) => {
     <SafeAreaViewFlex>
       <FlatList
         data={!orgaLoading ? listData : undefined}
-        renderItem={({ item }) => (
-          <OParlPreviewComponent data={item} key={item.id} navigation={navigation} />
-        )}
+        keyExtractor={(item, index) => `index${index}-id${item.id}`}
+        renderItem={({ item }) => <OParlPreviewComponent data={item} navigation={navigation} />}
         ListHeaderComponent={
-          <Wrapper style={styles.noPaddingTop}>
-            <DropdownSelect
-              data={dropdownData}
-              searchPlaceholder="Suche"
-              showSearch
-              searchInputStyle={styles.searchInput}
-              setData={setDropdownData}
-            />
-          </Wrapper>
+          dropdownData?.length <= 2 ? null : (
+            <Wrapper style={styles.noPaddingTop}>
+              <DropdownSelect
+                data={dropdownData}
```

Comment:
Can we use the `&&` operator instead of returning `null`?

```
dropdownData?.length <= 2 && (...)
```

Reply:
this could result in boolean `false` as value for `ListHeaderComponent`, which would be problematic i guess.